modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/spaces_navigation/SpacesNavigation.tsx

```diff
@@ -44,66 +61,89 @@ const SpacesNavigation: React.FC<SpacesNavigationProps> = ({
 		event.stopPropagation();
 	};
 
-	return (
-		<ClayPanel
-			collapsable
-			defaultExpanded
-			displayTitle={
-				<ClayPanel.Title className="align-items-center d-flex font-weight-semi-bold justify-content-between text-2 text-uppercase">
-					<span>{Liferay.Language.get('spaces')}</span>
+	const spacesNavigationItem = {
+		items: [
+			...assetLibraries.map(({name, settings, url}) => ({
+				href: url,
+				label: name,
+				sticker: {
+					displayType: settings?.logoColor as string,
+					label: name.charAt(0).toUpperCase(),
+				},
+			})),
+			{
+				href: allSpacesURL,
+				icon: 'box-container',
+				label: sub(
+					Liferay.Language.get('all-spaces-x'),
+					assetLibrariesCount
+				),
+			},
+		],
+		label: Liferay.Language.get('spaces'),
+	};
 
-					{showAddButton && (
-						<span className="float-right mr-2">
-							<ClayButtonWithIcon
-								aria-label={Liferay.Language.get('add-space')}
-								displayType="secondary"
-								onClick={onAddButtonClick}
-								size="xs"
-								symbol="plus"
-								title={Liferay.Language.get('add-space')}
-								type="button"
-							/>
-						</span>
-					)}
-				</ClayPanel.Title>
+	return (
+		<VerticalNav
+			aria-label="vertical navbar"
+			defaultExpandedKeys={new Set([Liferay.Language.get('spaces')])}
+			displayType="primary"
+			items={
+				showAddButton
+					? [
+							{
+								...spacesNavigationItem,
+								menubarAction: {
+									'aria-label':
+										Liferay.Language.get('add-space'),
+									'onClick': onAddButtonClick,
+									'role': 'menuitem',
+									'title': Liferay.Language.get('add-space'),
+								},
+							},
+						]
+					: [spacesNavigationItem]
 			}
-			showCollapseIcon
 		>
-			<ClayPanel.Body className="p-0">
-				<ul className="menubar-primary nav nav-stacked" role="menu">
-					{assetLibraries.map((assetLibrary) => (
-						<li className="nav-item" key={assetLibrary.id}>
-							<ClayLink
-								className="nav-link"
-								href={assetLibrary.url}
+			{(item: VerticalNavItem) => {
+				return (
+					<VerticalNav.Item
+						href={item.href}
+						items={item.items}
+						key={item.label}
+						menubarAction={
+							item.menubarAction as React.ComponentProps<
+								typeof ClayButtonWithIcon
 							>
-								<SpaceSticker
+						}
+						textValue={item.label}
+					>
+						<div className="autofit-row">
+							{item.sticker && (
+								<ClaySticker
```

Comment:
Here we can add `aria-hidden="true"` to prevent the screen reader from reading the sticker

https://github.com/user-attachments/assets/0c8cfa12-3b05-462c-ae53-714bbb013f2f